content/en/docs/concepts/cluster-administration/addons.md

```diff
@@ -5,6 +5,8 @@ content_type: concept
 
 <!-- overview -->
 
+{{% thirdparty-content %}}
+
```

Comment:
nit: If time, you could remove line 14 (Add-ons in each section are sorted alphabetically - the ordering does not imply any preferential status.)